docs/products/web-components/javascript-api.md

```diff
@@ -84,6 +116,204 @@ const entitlement = sesamy.getEntitlement("https://example.com/article", [
+    "cancelAt": 1687532004,
+    "cancelAtPeriodEnd": true,
+    "canceledAt": 1684853728,
+    "created": 1684853604,
+    "currentPeriodEnd": 1687532004,
+    "currentPeriodStart": 1684853604,
+    "endedAt": null,
+    "checkoutId": "test-3d31-40cc-bb71-9dfe08992e78",
+    "interval": "month",
+    "amount": 9900,
+    "status": "active",
+    "product": {
+      "name": "Montly subscription",
+      "description": " 99 kr per month w/taxes included",
+      "distributorId": "sesamy",
+      "optionId": "test-option-id",
+      "productId": "sid:test",
+      "sku": "sid:test",
+      "vendorId": "test-vendor",
+      "image": "https://images.sesamy.dev/cover/test/monthly-sub.png"
+    }
+  }
+]
+```
+
+## cancelSubscription
+
+The `cancelSubscription` function cancel a subscription from the current user:
+
+```javascript
+// Get subscriptions
+sesamy.cancelSubscription(SUBSCRIPTION_ID);
+```
+
+**Arguments:**
+
+- SUBSCRIPTION_ID (string): The id of the subscription to cancel.
+
+**Returns:**
+
+**Example:**
+
+```javascript
+// Cancel the subscription for the current user
+const subscriptions = await sesamy.cancelSubscription("test-id");
+```
+
+## getTransactions
+
+The `getTransactions` function queries for the transactions of the current user:
+
+```javascript
+// Get transactions
+sesamy.getTransactions();
+```
+
+**Arguments:**
+
+**Returns:**
+
+(Promise<[]\>): An array of the transactions
+
+**Example:**
+
+```javascript
+// Get the transactions for the current user
+const subscriptions = await sesamy.getTransactions();
+```
+
+The transactions will be in the following format
+
+```json
+[
+  {
+    "id": "test-id",
+    "checkKey": "test-aaa8-404d-89c8-checkkey",
+    "currency": "SEK",
+    "totalPrice": 9900,
+    "items": [
+      {
+        "period": {
+          "start": "2023-05-23T14:53:24.000Z",
+          "end": "2023-06-23T14:53:24.000Z"
+        },
+        "originalPrice": {
+          "market": "ES",
+          "total": 9900,
+          "totalTaxes": 381,
+          "discounts": [],
+          "netPrice": 9519,
+          "taxes": [
+            {
+              "amount": 381,
+              "taxId": "es-vat-news",
+              "taxPct": 4,
+              "taxAbbr": "IVA",
+              "taxName": "Impuesto sobre el Valor Añadido",
+              "base": 9519
+            }
+          ],
+          "totalDiscount": 0,
+          "currency": "SEK"
+        },
+        "distributorId": "sesamy",
+        "vendorId": "test-vendor",
+        "source": "STRIPE",
+        "purchaseOptionId": "test-option-id",
+        "title": "Monthly subscription",
+        "purchaseOptionType": "RECURRING",
+        "userId": "google-oauth2|test",
+        "cover": "https://images.sesamy.dev/cover/test/monthly-sub.png",
+        "geo": {
+          "zip": "08020",
+          "regionCode": "CT",
+          "userCountry": null,
+          "city": "Barcelona",
+          "countryCode": "ES",
+          "ip": "139.47.123.217",
```

Comment:
I don't think we want to show this. Should probably be filtered out by the api

Reply:
I think we are using the wrong query, should be '/me/transactions' instead of '/transactions', the new one is already trimmed and it includes the refunds and the receipt URL, which we'll probably want to add to the component at some point, or at least send it in the JS API, I'll create a ticket

Reply:
Here: https://github.com/sesamyab/web-components/pull/248

Reply:
Ignore these messages: the `/me/transactions` is not filtering by vendor 😓 Sorry for that, I'll need to modify the Wallet API to trim the results from `/transactions`.

Reply:
Or we can filter them in the component, what do you think?

Reply:
This is a breach of GDPR. We're not allowed to store the IP at all and definitely not expose it via an api. We need to make sure that no customer or partner can access any IP numbers. Ideally we should not store them.

Reply:
Got it, created this PR to fix it on the wallet: https://github.com/sesamyab/wallet-api/pull/234